src/app/components/MediaLoader/configs/aresMedia.ts

```diff
@@ -60,7 +60,8 @@ export default ({
   const versionID = versionsBlock?.versionId ?? '';
 
   const orientation =
-    ORIENTATION_MAPPING[versionsBlock?.types?.[0]] ??
+    (ORIENTATION_MAPPING[versionsBlock?.types?.[0]] ||
+      ORIENTATION_MAPPING[versionsBlock?.types?.[1]]) ??
```

Comment:
Could this be improved in any way with array destructuring & some clever checking whether the orientation is already in the mapping object?

```
const [orientationType] = versionsBlock?.types?.filter(type => Object.keys(ORIENTATION_MAPPING).includes(type));

const orientation = ORIENTATION_MAPPING[orientationType] || ORIENTATION_MAPPING.Original;
```

Reply:
Yeah this will definitely work, the only reason why we've targeted the specific positions in the types array is because I didn't know what the schema could be for types

Reply:
You could do a `find` instead of a `filter`... so that it would just match the first one it matches. But it's also OK to leave as is as like you say, we don't know what else might be in that array...

```
const orientationType = versionsBlock?.types?.find(type => Object.keys(ORIENTATION_MAPPING).includes(type));

const orientation = ORIENTATION_MAPPING[orientationType] || ORIENTATION_MAPPING.Original;
```

Reply:
I'll just switch it over to this as it's cleaner and is basically the same

Reply:
Ahh wait undefined can't be used as an array index selector

Reply:
Do you reckon something like this:

```js
const orientationType =
    versionsBlock?.types?.find(type =>
      Object.keys(ORIENTATION_MAPPING).includes(type),
    ) ?? 'Original';

  const orientation = ORIENTATION_MAPPING[orientationType];
```

Reply:
That would work!